libbeat/docs/upgrading.asciidoc

```diff
@@ -1,160 +1,216 @@
 [[upgrading]]
 == Upgrading
 
-This section gives general recommendations for upgrading the Beats:
+This section gives general recommendations for upgrading {beats} shippers:
 
 * <<upgrading-minor-versions>>
-* <<upgrading-5-to-6>>
+* <<upgrading-6-to-7>>
 * <<upgrade-mapping-template>>
 
 If you're upgrading other products in the stack, also read the
-{stack-ref}/index.html[Elastic Stack Installation and Upgrade Guide]. Want an
-upgrade list that's tailored to your stack? Try out our
-{upgrade_guide}[Interactive Upgrade Guide].
+{stack-ref}/index.html[Elastic Stack Installation and Upgrade Guide]. 
 
 [[upgrading-minor-versions]]
 === Upgrade between minor versions
 
-As a general rule, upgrading between minor versions (e.g. 6.x to 6.y, where x <
-y) can be done by simply installing the new release and restarting the Beat
-process, because the Beats keep backwards compatibility when it comes to
-configuration and exported fields. Please review the <<release-notes,release
-notes>> for potential exceptions.
+As a general rule, you can upgrade between minor versions (for example, 7.x to
+7.y, where x < y) by simply installing the new release and restarting the Beat
+process. {beats} typically maintain backwards compatibility for configuration
+settings and exported fields. Please review the
+<<release-notes,release notes>> for potential exceptions.
 
-Upgrading between non-consecutive major versions (e.g. 1.x to 6.x) is not
-supported. If you are on version 1.x of the Beats, we recommend first upgrading
-to 5.6.
+Upgrading between non-consecutive major versions (e.g. 5.x to 7.x) is not
+supported.
 
-[[upgrading-5-to-6]]
-=== Upgrade from 5.x to 6.x
+[[upgrading-6-to-7]]
+=== Upgrade from 6.x to 7.x
 
-Before upgrading your Beats, please review the <<breaking-changes, breaking
-changes>> doc.
+Before upgrading your {beats}, review the <<breaking-changes,breaking changes>>.
 
-// TODO: link to the stack upgrade docs
-//
-// If you are planning an upgrade of the full stack (Elasticsearch, Kibana, or
-// Logstash, in addition to Beats), please read the stack upgrade guide.
+If you're upgrading other products in the stack, also read the
+{stack-ref}/index.html[Elastic Stack Installation and Upgrade Guide]. 
+
+We recommend that you fully upgrade {es} and {kib} to version 7.0
+before upgrading {beats}. If you're on Beats 6.0 through 6.6,
+<<upgrading-to-6.7,upgrade to Beats 6.7>> *before* doing the {es} upgrade.
```

Comment:
Really minor, but when I was reading the generated doc, I saw this link and immediately thought to open it in a new tab because it looks like a link to a related resource. However, it's a link to the anchor tag a few lines down. Does it help the user that much or do we think that they'll find the content naturally and intuitively without the link? It might be worth removing it if so. Just a thought.

![image](https://user-images.githubusercontent.com/1238659/55265664-8683b180-5236-11e9-951d-d59aa42686b7.png)

Reply:
yah that's a valid point. I'll remove the link.